python/pyspark/context.py

```diff
@@ -240,6 +240,26 @@ def signal_handler(signal, frame):
         if isinstance(threading.current_thread(), threading._MainThread):
             signal.signal(signal.SIGINT, signal_handler)
 
+    def __repr__(self):
+        return "<SparkContext master={master} appName={appName}>".format(
+            master=self.master,
+            appName=self.appName
+        )
+
+    def _repr_html_(self):
+        return """
+        <div>
+            <p><b>Spark Context</b></p>
+            <ul>
+                <li>Spark <code>v{spark_version}</code></li>
+                <li><a href="{spark_ui_url}">Spark UI</a></li>
+            </ul>
+        </div>
+        """.format(
+            spark_version=self.version,
+            spark_ui_url=self.uiWebUrl,
```

Comment:
Thanks for the explanation @rgbkrk. That make sense. The reason for asking is method _repr_ which you have added don't have the "," at the end.